raiden_contracts/constants.py

```diff
@@ -1,7 +1,7 @@
 from enum import Enum, IntEnum
 
 # Do not change this, this is handled by bumpversion with .bumpversion_contracts.cfg
-CONTRACTS_VERSION = "0.7.0"
+CONTRACTS_VERSION = "0.8._"
```

Comment:
Out of curiosity... what does the underscore mean in this case?

Reply:
The message is "This is something similar to `0.3._`".

Reply:
Hehe ... the only reason `0.3._` had an underscore for the patch part of the version was to signify that there is no patch version and that each change bumps the minor version. Or something silly like that. No need to repeat that now.

Reply:
I'm intentionally repeating this.  The hidden message is "This is something as ad-hoc as `0.3._`.  Don't take it seriously (or deploy it on the mainnet)".

Reply:
This convention won't relay the message. Also there is no technical argument for having this version numbering scheme so i would go for using 0.8.0

Reply:
No that's too confusing because `0.8.0` looks normal.  The version in this PR is very different from other versions (not deployable on the mainnet).  It almost deserves a separate PyPI package `raiden-contract-for-testing-only`.

If you want a normal version scheme, wait until [the proper solution](https://github.com/raiden-network/raiden-contracts/issues/599#issuecomment-466200340) lands.  It's up to you.

Reply:
@rakanalh But tell me your choice, if I should push this PR or go implement the proper solution ^.  Currently I'm pushing this because this is the ASAP solution that you wanted.

Reply:
I'm accommodating the abnormal situation, where the bytecode for the mainnet is not tested, but something else is being tested.  And I chose to provide something else here ASAP.  But the unlimited version is something that does not really belong the Ithaca version of `raiden-contracts`.  My opinions aside, I'm taking extra steps because keeping the CI running in `raiden` repo is important on its own.

Reply:
@rakanalh if it's just about names.  Instead of `0.8._` I can also say `0.8-unlimited-only-for-testing`.  Is this fine? I try.

Reply:
No, sorry the train leaves in 15min.  And I cannot change the version string, compile the sources, deploy, and talk to Etherscan.  I give up this PR as it is now, for this week.

Reply:
On my side, `0.8._` does not need to convey the meaning.  It just needs to look weird, and cause questions like "what does it mean"?

Reply:
@pirapira In the end it's your choice. Will there ever be a `0.8.0`? Or using the underscore effectively uses up `0.8.0` too?  As described above `0.3._` was just something we did back then. I don't even remember the exact reason so blindly copying it sounds dirty.

If you want the `patch` part of the version being an `_` meaning that the version will be unlimited then that's also your choice. But this should be documented and be offered as part of the interface between raiden contracts and client since we will probably be stuck with `0.8._` for a while.

I like the approach you linked above as the [proper way](https://github.com/raiden-network/raiden-contracts/issues/599#issuecomment-466200340).

How much time would that take? @rakanalh can you wait until that? If not and this satisfies the requirements to get your PR green then no need to analyze it further, just merge it.

Reply:
I think the solution in this PR is dirty, and it deserves a dirty name.

I documented `0.8._` in README.

There will not be `0.8.0`.  The next change will be putting back the limit, and that chages the ABI, so it ill be `0.9.0`.

The proper way, I estimate till Tuesday next week (very optimistic, but achievable if I cut all meetings and any distractions).

Reply:
I would wait for the "proper" way as well.

Reply:
That's a sane approach!